### ISSUE TITLE
Remove Infinispan 12 workarounds

### ISSUE THREAD

**Issue by hmlnarik** [keycloak/keycloak#10924]
### Description

Commit 74581b5 introduced workarounds for Infinispan 12. These should be removed once Infinispan is updated

There are changes in #10917 in testsuite/model/pom.xml which should be removed as well.

Depends on #10112.

**Comment by stianst** [comment]
This needs to wait until Keycloak 19 as EAP 7.4 uses Infinispan 11

**Comment by martin-kanis** [comment]
@ahus1 @mhajas Is this duplicate of https://github.com/keycloak/keycloak/issues/13962?
